DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -284,6 +351,70 @@ void L1TStage2uGT::analyze(const edm::Event& evt, const edm::EventSetup& evtSetu
      } // selecting LastCollisionInTrain
   } // end of uGTAlgs = 1
 
+
+ for(auto itr = uGtAlgs->begin(0); itr != uGtAlgs->end(0); ++itr) {
```

Comment:
Can these be integrated in the existing for loops? At least the first two loops are common as far as I can see.